ipl/cfuncs/icall.h

```diff
@@ -170,6 +174,7 @@
 #define Type(d) ((int)((d).dword & TypeMask))
 
 typedef long word;
+typedef unsigned long uword;
```

Comment:
The way we are handling this type (size) has the potential of breaking things in unexpected ways on some platforms. We probably should be consistent with what the  runtime expect:

https://github.com/uniconproject/unicon/blob/master/src/h/typedefs.h#L18